opentelemetry-tracing/opentelemetry-python-tracing/falcon/main.py

```diff
@@ -1,30 +1,23 @@
 #!/usr/bin/env python
 
 import json
-import sys
-import logging
 
 import falcon
 from wsgiref import simple_server
-from opentelemetry import trace
-from opentelemetry.trace import SpanKind
 
 
-tracer = trace.get_tracer(__name__)
-
 class HelloWorldResource(object):
     def on_get(self, req, resp):
         """Handles GET requests"""
-        with tracer.start_as_current_span('internal_span', kind=SpanKind.INTERNAL) as span:
-            resp.body = json.dumps({'ok': True})
+        resp.body = json.dumps({"ok": True})
 
 
-app = falcon.API()
+app = falcon.App()
```

Comment:
This change plus version update should be enough. Any reason to remove tracing or change the quotation marks?

Reply:
Hey,
I'll change back to single quotes 👍🏻 

I find it very confusing that Falcon is the only tracing example with a manual trace. 
This repo tends to supply examples of how to instrument different frameworks. Using the SDK to start a trace manually has nothing to do with this framework or the other.